frontend/test/specs/post/timelineDirectiveSpec.js

```diff
@@ -0,0 +1,158 @@
+        state: 'active',
+        permissions: {}
+    };
+
+    var post = {
+        title: 'post principal',
+        author_key: user.key,
+        institution_key: "institution_key",
+        key: "123456",
+        state: 'published'
+    };
+
+    var post_with_activity = {
+        title: 'post principal',
+        author_key: user.key,
+        institution_key: "institution_key",
+        key: "123456",
+        state: 'published',
+        comments: "/api/posts/123456/comments",
+        number_of_comments: 1
+    };
+
+    var options = [{'id' : 0,
+                    'text': 'Option number 1',
+                    'number_votes': 0,
+                    'voters': [] },
+                    {'id': 1,
+                    'text': 'Option number 2',
+                    'number_votes': 0,
+                    'voters': [] }];
+
+    var survey = { 'title' : 'The Survey',
+                    'type_survey' : 'multiple_choice',
+                    'options' : options,
+                    'key': '654321'
+                    };
+
+    var content = {
+        'scrollTop': 1,
+        'offsetHeight': 3,
+        'scrollHeight': 5
+    }
+
+    var posts = [survey, post];
+
+    beforeEach(inject(function($controller, AuthService, $rootScope) {
+        rootScope = $rootScope;
+        AuthService.login(user);
+
+        spyOn(document, 'getElementById').and.callFake(function () {
+            return content
+            
+        });
+
+        spyOn(Utils, 'setScrollListener').and.callThrough();
+
+        createController = function(){
+            return $controller('TimelineController', {
+                $rootScope: rootScope
+            });
+        }
+
+        timelineCtrl = createController();
+        timelineCtrl.content = content;
+        timelineCtrl.user = new User(user);
+        timelineCtrl.posts = posts;
+    }));
+
+    describe('Should create observer', function() {
+        it('should call root scope to create observer', function() {
+            spyOn(rootScope, '$on').and.callThrough();
+            timelineCtrl = createController();
+            expect(rootScope.$on).toHaveBeenCalled();
+        });
+    });
+
+    describe('Should call loadMorePosts', function() {
+        it('should set variable testLoadMorePosts to true.' +
+            ' This variable was create to test if function that was' +
+            ' passed through bind Controller is called', function() {
+
+            var callbackLoadMorePost = function(){
+                return {
+                    then : function(){
+                        timelineCtrl.testLoadMorePosts = true;
+                        return timelineCtrl.testLoadMorePosts
+                    }
+                }
+            };
+            expect(timelineCtrl.testLoadMorePosts).toEqual(undefined);
+            
+            timelineCtrl = createController();
+            timelineCtrl.loadMorePosts = callbackLoadMorePost;
+            timelineCtrl.content = content;
```

Comment:
this variable is already being created in the first beforeeach.

Reply:
quando inicializa um o controller novamente ele se perde.